Sprint-1/interpret/time-format.js

```diff
@@ -12,13 +12,29 @@ console.log(result);
 // For the piece of code above, read the code and then answer the following questions
 
 // a) How many variable declarations are there in this program?
+//There are 5 variable declarations: const movieLength,const remainingSeconds,const totalMinutes,const remainingMinutes,const totalHours
+
 
 // b) How many function calls are there?
+//There are 0 function calls in this program. 
+//The expressions involve arithmetic operations, but no functions 
```

Comment:
You're right that most of the logic here is done without using functions. But what about the last part, where we log the result? Is there a function call involved here?

Reply:
There are 1 function calls in this program which is console.log()